src/main/java/com/csv/communitytrackerjava/service/ProjectServiceImpl.java

```diff
@@ -76,14 +79,19 @@ public ProjectResponseDTO updateProject(ProjectUpdateDTO projectUpdateDTO, Integ
 
         projectFound.setProjectCode(projectUpdateDTO.getProjectCode());
         modelMapper.map(projectUpdateDTO, projectFound);
-        payloadDTO.setAdditionalProperty("projects", projectMapper.toDTO(projectRepository.save(projectFound)));
+        payloadDTO.setAdditionalProperty("projects", projectMapper
+                .toDTO(projectRepository.save(projectFound)));
         return toProjectResponseDTO("Successfully update project.", payloadDTO);
     }
 
     @Override
-    public ProjectResponseDTO findAllProject() {
+    public ProjectResponseDTO findAllProject(Pageable pageable) {
         ProjectPayloadDTO payloadDTO = new ProjectPayloadDTO();
-        payloadDTO.setAdditionalProperty("projects", projectRepository.findAll());
+        List<ProjectDTO> projectList = (
+                projectRepository.findByIsActiveTrue(pageable)
```

Comment:
Do we really want to get only active project here?

Reply:
yes for the testing of the delete.